portfolio/src/main/webapp/script.js

```diff
@@ -17,12 +17,59 @@
  */
 function addRandomGreeting() {
   const greetings =
-      ['I\'m ready, I\'m ready, I\'m ready...', 'You know what\'s funnier than 24? 25!', 'East?! I thought you said Weest!', 'Squidwaard, I used your clarinet to unclog my toilet!'];
+      ['I\'m ready, I\'m ready, I\'m ready...', 
+      'You know what\'s funnier than 24? 25!', 
+      'East?! I thought you said Weest!', 
+      'Squidwaard, I used your clarinet to unclog my toilet!'];
 
   // Pick a random greeting.
   const greeting = greetings[Math.floor(Math.random() * greetings.length)];
 
   // Add it to the page.
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
+}
+
+/**
+ * Reveal hidden description upon mouseover and hides the project title
+   text.
+ * @param {object} hoveredItem An anchor tag containing the
+      project image and text
+ */
+function revealOnMouseOver(hoveredItem) {
+  let toReveal = hoveredItem.getElementsByClassName('hidden')[0];
+  let toHide = hoveredItem.getElementsByClassName('visible')[0];
+
+  let background = hoveredItem.getElementsByClassName('background');
+
+  toReveal.style.visibility = 'visible';
+  toHide.style.visibility = 'hidden';
+
+  // loop through all the tags that make up the background image
+  // and lowers its brightness
+  for (i = 0; i < background.length; i++) {
```

Comment:
It is recommended to use a for-of loop.

See: https://google.github.io/styleguide/jsguide.html#features-for-loops

Reply:
Done!